src/runner.rs

```diff
@@ -1278,21 +1198,15 @@ impl FunctionContext {
 		self.is_initialized
 	}
 
-	pub fn initialize(
-		&mut self,
-		locals: &[Local],
-		value_stack: &mut ValueStack,
-	) -> Result<(), TrapKind> {
+	fn initialize(&mut self, locals: &[Local], value_stack: &mut ValueStack) -> Result<(), TrapKind> {
```

Comment:
Why does this have to be private?

Reply:
ValueStack is private so it can't be part of a public interface.